packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -83,7 +83,6 @@ contract OPContractsManager is ISemver, Initializable {
         FaultDisputeGame faultDisputeGame;
         PermissionedDisputeGame permissionedDisputeGame;
         DelayedWETH delayedWETHPermissionedGameProxy;
-        DelayedWETH delayedWETHPermissionlessGameProxy;
```

Comment:
Let's keep this in the struct, that way the interface stays the same when we add it back later. Similar to how we have `faultDisputeGame` here even though it's not deployed

Reply:
https://github.com/ethereum-optimism/optimism/pull/12156/commits/e10ee3cc774be222566781a5bc024ca0f65f77bb